client/my-sites/plans-features-main/index.tsx

```diff
@@ -461,13 +462,20 @@ const PlansFeaturesMain = ( {
 		_customerType = 'business';
 	}
 
-	const filteredDisplayedIntervals = useFilteredDisplayedIntervals( {
+	let filteredDisplayedIntervals = useFilteredDisplayedIntervals( {
```

Comment:
Is there anything blocking from importing and applying the restriction inside `useFilteredDisplayedIntervals`? Ideally no need to mutate the structure further here.

Reply:
Good idea! Thanks!
I moved the filtering to the `useFilteredDisplayedIntervals` hook.